src/inc/cordebug.idl

```diff
@@ -274,10 +274,10 @@ interface ICorDebugDataTarget : IUnknown
         CORDB_PLATFORM_WINDOWS_X86,       // Windows on Intel x86
         CORDB_PLATFORM_WINDOWS_AMD64,     // Windows x64 (Amd64, Intel EM64T)
         CORDB_PLATFORM_WINDOWS_IA64,      // Windows on Intel IA-64
-        CORDB_PLATFORM_MAC_PPC,           // MacOS on PowerPC
-        CORDB_PLATFORM_MAC_X86,           // MacOS on Intel x86
+        CORDB_PLATFORM_UNIX_PPC,          // Posix supporting OS on PowerPC
+        CORDB_PLATFORM_UNIX_X86,          // Posix supporting OS on Intel x86
         CORDB_PLATFORM_WINDOWS_ARM,       // Windows on ARM
-        CORDB_PLATFORM_MAC_AMD64,         // MacOS on Intel x64 
+        CORDB_PLATFORM_UNIX_AMD64,        // Posix supporting OS on Intel x64 
         CORDB_PLATFORM_WINDOWS_ARM64      // Windows on ARM64
```

Comment:
These are publicly documented values that ship in public SDK. Renaming them is breaking change - random things out there may depend on. It would be safer to leave the existing ones alone - add obsolete/do not use comment next to them, and add the new ones at the end: the new Unix ones should not have PPC, but you can add ARM64 while you are on it.